src/mapblock_mesh.cpp

```diff
@@ -603,7 +603,7 @@ static void makeFastFace(TileSpec tile, u16 li0, u16 li1, u16 li2, u16 li3,
 		core::vector2d<f32>(x0 + w * abs_scale, y0) };
 
 	for (int layernum = 0; layernum < MAX_TILE_LAYERS; layernum++) {
-		TileLayer *layer = &tile.layers[layernum];
+		const TileLayer *layer = &tile.layers[layernum];
```

Comment:
i need to change if because TileSpec tile is now const TileSpec &, then the pointer should be const (required by compiler)